testprojects/tests/python/pants/dummies/test_with_thirdparty_dep.py

```diff
@@ -0,0 +1,11 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from future.utils import text_type
+
+
+def test_f():
+  assert isinstance(text_type("foo"), text_type)
```

Comment:
Because we have unicode literals, I recommend simplifying this to
```suggestion
  assert isinstance("foo", text_type)
```

Even better
```suggestion
from future.utils import str

  assert isinstance("foo", str)
```